src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
-    {
-        ushort ws_row;
-        ushort ws_col;
-        ushort ws_xpixel;
-        ushort ws_ypixel;
-    }
+    import core.sys.posix.sys.ttycom; // Terminal related ioctls
```

Comment:
This might want to be a public import. In C, would you just write `#include<sys/ioctl.h>` or would you explicitly do `#include<sys/ttycom.h>`? If it is the former, you'd need to public import in D so the file is exposed here too.